tokio/src/runtime/task/trace/tree.rs

```diff
@@ -0,0 +1,126 @@
+use std::collections::{hash_map::DefaultHasher, HashMap as Map, HashSet as Set};
```

Comment:
We tend only to rename imports if there is a name conflict. I was reading the code and did not recognize the `Map` and `Set` types. `HashSet` and `HashMap` let me guess the type without looking at the import list.